src/ResourceManager/Profile/Commands.Profile/packages.config

```diff
@@ -10,23 +10,23 @@
   <package id="Microsoft.Azure.Management.Authorization" version="2.0.0" targetFramework="net45" />
   <package id="Microsoft.Azure.Management.ResourceManager" version="1.9.0-preview" targetFramework="net452" />
   <package id="Microsoft.Azure.Management.Resources" version="2.20.1-preview" targetFramework="net452" />
-  <package id="Microsoft.Azure.PowerShell.Aks" version="1.1.7-preview" targetFramework="net452" />
-  <package id="Microsoft.Azure.PowerShell.Authentication" version="1.1.7-preview" targetFramework="net452" />
-  <package id="Microsoft.Azure.PowerShell.Authentication.Abstractions" version="1.1.7-preview" targetFramework="net452" />
-  <package id="Microsoft.Azure.PowerShell.Authentication.ResourceManager" version="1.1.7-preview" targetFramework="net452" />
-  <package id="Microsoft.Azure.PowerShell.Authorization" version="1.1.7-preview" targetFramework="net452" />
-  <package id="Microsoft.Azure.PowerShell.Common" version="1.1.7-preview" targetFramework="net452" />
-  <package id="Microsoft.Azure.PowerShell.Compute" version="1.1.7-preview" targetFramework="net452" />
-  <package id="Microsoft.Azure.PowerShell.Graph.Rbac" version="1.1.7-preview" targetFramework="net452" />
-  <package id="Microsoft.Azure.PowerShell.KeyVault" version="1.1.7-preview" targetFramework="net452" />
-  <package id="Microsoft.Azure.PowerShell.Network" version="1.1.7-preview" targetFramework="net452" />
-  <package id="Microsoft.Azure.PowerShell.ResourceManager" version="1.1.7-preview" targetFramework="net452" />
-  <package id="Microsoft.Azure.PowerShell.ScenarioTest.ResourceManager" version="1.1.7-preview" targetFramework="net452" />
-  <package id="Microsoft.Azure.PowerShell.ServiceManagement" version="1.1.7-preview" targetFramework="net452" />
-  <package id="Microsoft.Azure.PowerShell.Storage" version="1.1.7-preview" targetFramework="net452" />
-  <package id="Microsoft.Azure.PowerShell.Storage.Management" version="1.1.7-preview" targetFramework="net452" />
-  <package id="Microsoft.Azure.PowerShell.Strategies" version="1.1.7-preview" targetFramework="net452" />
-  <package id="Microsoft.Azure.PowerShell.Websites" version="1.1.7-preview" targetFramework="net452" />
+  <package id="Microsoft.Azure.PowerShell.Aks" version="1.0.213-preview" targetFramework="net452" />
```

Comment:
will need to republish and update

Reply:
@cormacpayne Just need to republish common code, otherwise LGTM.  Marking as do not merge until commoon code is republished.